validator/testcases/content.py

```diff
@@ -157,7 +157,7 @@ def test_packed_packages(err, xpi_package=None):
                 err.notice(
                     err_id=('testcases_content',
                             'test_packed_packages',
-                            'blacklisted_js_library'),
+                            'denied_js_library'),
```

Comment:
Well that's just slightly confusing. Thanks I'll go back and actually read this instead of doing a quick search and replace.